virttest/utils_2to3_wrap.py

```diff
@@ -0,0 +1,14 @@
+"""
+Port Python2 to Python3 utility functions.
+Please make sure do NOT import other modules from virttest
+
+"""
+from six import string_types
+
+
+def decode_to_text(stream, encoding='utf-8'):
```

Comment:
suggest to hard code encoding to uft-8, but use encoding = sys.getdefaultencoding()

Reply:
Updated